roadmap/propagate-rust-panic-through-native-frame.md

```diff
@@ -46,3 +48,37 @@ cases (contains destructors, no affordances). We will also have to
 specify this on a per-target basis, as the details will vary depending
 on what exception mechanism is in use.
```

Comment:
This paragraph and the bullet points at the end, ignore the fact that native frames can be written in Rust. That is, the call graph above can be Rust->Rust->Rust, and this then specifies that unwinding with a Rust panic through Rust is UB. 

The bullet points at the end of the file also ignore this, making things like the behavior of a Rust panic crossing a Rust thread boundary undefined.

Reply:
"Native" is intended to mean "not Rust". I'll go through and clarify that; thank you.